src/app/modules/project/highlights-modal/highlights-modal.component.scss

```diff
@@ -0,0 +1,74 @@
+@import "assets/styles/variables";
+
+.highligts-modal-container {
+  padding: 35px;
+  z-index: 1151;
+  position: relative;
+  top: 100px;
+  left: 50%;
+  transform: translateX(-50%);
+  background-color: #fff;
+  min-width: 100%;
+
+  .heading-row {
+    padding: 0.75rem 0;
+  }
+  .highlight-wrapper {
+    .row {
+      min-height: 54px;
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  .highlight-wrapper:hover {
+    background-color: #f8f9fa;
+    cursor: pointer;
+  }
+
+  .list-group-item {
+    border-left: none;
+    border-right: none;
+    cursor: pointer;
+  }
+
+  .delete {
+    padding-right: 10px;
+    display: flex;
+    justify-content: flex-end;
+  }
+  .delete img:hover {
+    opacity: 0.7;
+  }
+
+  .add-button {
+    display: flex!important;
+    justify-content: center;
+    align-items: center;
+    span {
+      vertical-align: bottom;
+      line-height: 30px;
+    }
+    img{
+      height: 25px;
+      width: auto;
+    }
+  }
+  .trash-icon{
+    color: $light-mode-grey-secondary;
```

Comment:
```suggestion
    color: $light-mode-grey-secondary;
    
    &:hover {
      color: $accent-color-red-secondary;
    }
```

Reply:
Adds a small hover over color to let the user know that it's a button they can click.

Reply:
> Another thing I noticed, is when I click 2 times on the save button rapidly, the highlight is added twice. I don't know if this is really a big problem, but I want to validate that this is a known occurrence.

@Brend-Smits debounce click fixes this issue that @Ruby77 mentioned in the discussion above

Reply:
Oh I see, that's pretty cool. Thanks.